Falcon/Falcon/Physics/vehicle/Vehicle.cpp

```diff
@@ -286,7 +348,23 @@ namespace physics
 				if(car!=nullptr)
 					fmemory::fdelete<Car>(car);
 			}
+
+			for (CarController* controller : gCarControllers)
+			{
+				if (controller != nullptr)
+					fmemory::fdelete<CarController>(controller);
+			}
 		}
 
+		CarController* GetCarContoller(Car* car)
```

Comment:
There's a warning here cause the function doesn't always return a value. After printing the error in the else, you could also return nullptr and I think that would fix the warning